### ISSUE TITLE
Steam Big Picture Mode Overlay

### ISSUE THREAD

**Issue by joseph881** [Kaldaien/TSF#4]
Not sure if this an actual issue or just my lack of knowledge in configuring the mod, but normally (without the mod), when the game is loaded through big picture mode, the steam overlay corresponds to it and lets you open it via the controller's guide button, but with this mod enabled, the game only loads the regular steam overlay regardless of which mode it is launched from.

I've tested this by installing and reinstalling the mod. It's not a major issue, and certainly something worth giving up in exchange for how great the game looks and runs now, but just throwing it out there as I haven't seen or read anything regarding this.

Thanks for your work, it is greatly appreciated.

**Comment by Kaldaien** [comment]
I cannot reproduce this problem :-\ I use Big Picture mode all the time because it's the only way to configure my Steam Controller.

Do you still have TOS.exe set to run as administrator? I'm thinking maybe something like a privilege elevation prompt is confusing Steam for you.

**Comment by joseph881** [comment]
That's weird. I do have both exes set to run as admin. I can also simulate
the problem by setting TOS.exe to run in Windows XP compatibility mode with
the mod not installed (did this back to try to get 40FPS). I can't find it
to be a privilege issue because since UAC was introduced in WinVista, I've
always turned it off. I will try to take a video showing the problem when I
get home, but in the meantime, I will list the specs of the pc I'm using to
play.

Windows 10 Pro 64bit
Intel D77GAL-70K Mobo
Intel Core i7 3770K CPU
16GB (2x8) G.SKILL Ripjaws X DDR3 @ 2133MHz
Nvidia GForce GTX970 w/ the latest Driver (362.00)
Xbox One Controller via the USB Wireless Adapter

**Comment by joseph881** [comment]
Nevermind, I figured it out. In my setup, the Run as Administrator option
was causing that problem instead of solving any other. I'm guessing because
my UAC is off and things I run should be elevated anyway.

I installed 0.7.0, deleted the old, remnant tsfix.exe, and unchecked the
Run as Administrator checkbox for TOS.exe and now mod and steam big picture
mode overlay work perfectly fine.

If UAC is off, users don't need to force it to run as admin.
